### PR TITLE
app: replace React

### LINKED ISSUE #10: Undesired perceptible start time of React
It's not possible to press a shortcut and immediately start typing e.g. Ctrl/Cmd+Shift+J for new task. Window opens about half a second later which is bad UX.

1. maybe we can try to switch to preact having better starting times
1. maybe just use plain js (still not that much to be refactored if we do it soon)

### PR DESCRIPTION
**Pull request by DaAitch**
- replace React with Preact
- use parcel bundler
- performance improvements: e.g. don't close, but hide TaskList

Fixes: #10

### CODE REVIEW

**Comment by DaAitch** [comment]
@s1hofmann thanks for merging #8. I resolved the merge conflicts so this can now be reviewed.
almost everything stays the same, except: `npm run preact-dev` :)

File: package.json [hunk 1]
```diff
@@ -2,37 +2,44 @@
...
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
-    "react-dev": "BROWSER=none npm start",
-    "electron-dev": "nodemon --watch public/electron.js node_modules/.bin/electron .",
-    "all-dev": "concurrently \"BROWSER=none npm start\" \"wait-on http://localhost:3000 && electron .\"",
-    "electron-pack": "build build/electron.js",
+    "start": "npm run all-dev",
+    "build": "parcel build ./src/index.html ./src/screenshot.html --out-dir=build --public-url ./",
+    "all-dev": "concurrently \"BROWSER=none npm run preact-dev\" \"wait-on http://localhost:3000 && electron .\"",
```
Comment (s1hofmann) [inline package.json]: On my machine, running `npm run all-dev` only works the first time. Subsequent runs do not start the electron application properly.
Running `concurrently \"electron . && wait-on http://localhost:3000\" \"BROWSER=none npm run preact-dev\"` solves this behaviour and starts the application properly.

Comment (DaAitch) [reply package.json]: didn't use all-dev for a while can confirm it's not working anymore. problem is, that we now have two entry points and `http://localhost:3000` is not correct, but `http://localhost:3000/index.html` is. Fixed that, please check.

Comment (s1hofmann) [reply package.json]: 👍 All good now!

**Pull request merged**